src/main/java/hudson/plugins/promoted_builds/conditions/UpstreamPromotionCondition.java

```diff
@@ -87,6 +91,13 @@ public void add(String promotion) {
 
     @Extension
     public static final class DescriptorImpl extends PromotionConditionDescriptor {
+        public boolean isApplicable(@Nonnull Job<?, ?> item, @Nonnull TaskListener listener) {
+            if(item instanceof AbstractProject){
+                return isApplicable((AbstractProject)item, TaskListener.NULL);
+            }
+            return true;
+        }
+
         public boolean isApplicable(AbstractProject<?,?> item) {
```

Comment:
remove? Default implementation should be doing that

Reply:
@oleg-nenashev ok i should be changing `return isApplicable` to just `isApplicable`. would that be good?!